CommandLineClient/src/main/java/com/dslplatform/compiler/client/Utils.java

```diff
@@ -187,8 +188,8 @@ public CommandResult(final String output, final String error, final int exitCode
 		private final StringBuilder output = new StringBuilder();
 		private IOException exception;
 
-		private ConsumeStream(final InputStream stream, final Context context) {
-			this.reader = new BufferedReader(new InputStreamReader(stream));
+		private ConsumeStream(final InputStream stream, final Context context, final Charset charset) {
+			this.reader = new BufferedReader(new InputStreamReader(stream, charset));
```

Comment:
:(
.NET is somewhat smarter about it. If you don't specify the encoding it will try to "detect" it. Not just use the default encoding.